### PR TITLE
paths: compatibility for derivation paths checks

### PR DESCRIPTION
**Pull request by jpochyla**

### CODE REVIEW

**Comment by tsusanka** [comment]
I forgot to test against mcu :(. I'll look into it

File: trezorlib/tests/device_tests/test_msg_signtx.py [hunk 1]
```diff
@@ -87,10 +95,12 @@ def btn(code):
...
 
     for i, inp in enumerate(inputs):
-        expected_responses += [
-            tx_request(t.TXINPUT, request_index=i),
-            tx_request(t.TXMETA, tx_hash=inp.prev_hash),
-        ]
+        expected_responses.append(tx_request(t.TXINPUT, request_index=i))
+        if unknown_path and TREZOR_VERSION != 1:
+            expected_responses.append(
+                btn(proto.ButtonRequestType.UnknownDerivationPath)
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_signtx.py]: `btn(b.UnknownDerivationPath)`

File: trezorlib/tests/device_tests/test_multisig_change.py [hunk 2]
```diff
@@ -135,7 +136,16 @@ def _responses(self, inp1, inp2, change=0):
             proto.TxRequest(
                 request_type=proto.RequestType.TXINPUT,
                 details=proto.TxRequestDetailsType(request_index=0),
-            ),
+            )
+        ]
+        if (
+            TREZOR_VERSION != 1
+        ):  # trezor 1 does not have UnknownDerivationPath implemented
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_multisig_change.py]: please move the comment to the next line, then black will not make this particular mess :)

File: trezorlib/tests/device_tests/test_multisig_change.py [hunk 3]
```diff
@@ -162,6 +172,15 @@ def _responses(self, inp1, inp2, change=0):
                 request_type=proto.RequestType.TXINPUT,
                 details=proto.TxRequestDetailsType(request_index=1),
             ),
+        ]
+        if (
+            TREZOR_VERSION != 1
+        ):  # trezor 1 does not have UnknownDerivationPath implemented
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_multisig_change.py]: same

**Comment by tsusanka** [comment]
So this should be ready. I'll look into the segwit multisig soon, but this is good to go from my side.

**Comment by matejcik** [comment]
looking good :+1:

**Pull request merged**